app/src/main/java/com/intervaltimer/google/activities/WorkoutActivity.java

```diff
@@ -96,6 +100,13 @@ protected void onCreate(Bundle savedInstanceState) {
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
+        Intent receivedIntent = getIntent();
+        if (receivedIntent != null && receivedIntent.getData() != null) {
+            Log.i(WORKOUT_INTENT_TAG, "Intent called, but no active workout");
```

Comment:
If I read this correct, we are in a situation where we can't fullfil the user intent because it the app is in another state? Should we give the user a hint about this?

Reply:
Made a toast to appear.